connect-inject/endpoints_controller_test.go

```diff
@@ -560,6 +560,7 @@ func TestReconcileCreateEndpoint(t *testing.T) {
 						DestinationServiceID:   "pod1-service-created",
 						LocalServiceAddress:    "",
 						LocalServicePort:       0,
+						TransparentProxy:       &api.TransparentProxyConfig{},
```

Comment:
Are these necessary if empty?

Reply:
Unfortunately, not in this case as it expects a pointer. An empty value would equate to a `nil` and not an empty field. this should be fixed in the future and we should be able to revert this change when Consul returns a nil value when provided a nil value.

Reply:
Ah sweet, I wasn't sure if this fell under that case, thanks for the explanation!!